### ISSUE TITLE
config files for training cifar10

### ISSUE THREAD

**Issue by Chen-Cai-OSU** [EmilienDupont/augmented-neural-odes#6]
Hello,

Thank you very much for the nice work. I am playing around your code and was wondering would it be possible for you to share the config files to train the NODE or ANODE on cifar10? I am able to get good accuracy on mnist, but for cifar using the same configs as you mentioned [here](https://github.com/EmilienDupont/augmented-neural-odes/issues/2), I can only get around 55%. Is that normal?

**Comment by Chen-Cai-OSU** [comment]
I see in the paper you got ~60% accuracy on cifar10 using ANODE. Is there any intuition why the accuracy is that low? Doesn't augment the input space can eliminate the restriction of flow induced by neural ODE? What do you think the main bottleneck here? Thank you.

**Comment by EmilienDupont** [comment]
Hi, sorry for the slow reply, I was working on an ICML submission, so I finally have time to catch up on everything now!

This is the config I used for creating the final CIFAR10 accuracy results for the paper:

```json
{
  "id": "cifar_acc_final", 
  "num_reps": 5, 
  "dataset": "cifar10", 
  "model_configs": [
    {"type": "anode", 
     "num_filters": 64, 
     "augment_dim": 10, 
     "time_dependent": true, 
     "lr": 0.001, 
     "non_linearity": "relu", 
     "weight_decay": 0.0, 
     "validation": true}
   ], 
  "training_config": {"batch_size": 256, "record_freq": 10, "print_freq": 10, "epochs": 40}
}
```

It looks like the difference is the number of augmented dimensions. In the config you refer to `augment_dim=5` whereas in this config `augment_dim=10`. If you rerun the experiments with this config you should hopefully get ~60% accuracy.

In terms of what the main bottleneck is here, it is not completely clear. NODE-like models typically have worse performance than ResNets even when using augmented dimensions. There could be several reasons for this, for example the way the weights are parameterized in different layers of the ODE could be suboptimal (i.e. in ResNets each set of weights is independent, whereas for a NODE the weights at every timestep are the same and the only difference is made through the time parameter) or the gradient that arises from the adjoint method may not be as useful for learning as one from a regular neural net. However, I am not sure exactly why this performance gap exists and I hope there will be a paper soon trying to analyse this in more detail!

**Comment by Chen-Cai-OSU** [comment]
Thank you very much!
